src/aiovodafone/models/technicolor.py

```
@@ -0,0 +1,447 @@
+                    retries,
+                )
+                # sleep for 2 seconds, just like the dashboard does
+                await asyncio.sleep(2)
+            except ClientResponseError:
+                _LOGGER.exception("Failed to retrieve '%s' results", key)
+
+        raise ResultTimeoutError(
+            "'%s' results not available after %d retries",
+            key,
+            retries,
+        )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)